locales/fr/pokemon.json

```
@@ -333,7 +333,7 @@
   "Magikarp Calico (Orange, White, Black)": "[[Magikarp]] [[alt.karp.calico]] (Rouge-Blanc-Noir)",
   "Magikarp Calico (White, Orange)": "[[Magikarp]] [[alt.karp.calico]] (Blanc-Rouge)",
   "Magikarp Calico (Orange, Gold)": "[[Magikarp]] [[alt.karp.calico]] (Rouge-Or)",
-  "Magikarp Orange Two Tone": "[[Magikarp]] [[alt.karp.twoTone]] [[alt.karp.orange]]",
+  "Magikarp Orange Two-Tone": "[[Magikarp]] [[alt.karp.twoTone]] [[alt.karp.orange]]",
   "Magikarp Orange Orca": "[[Magikarp]] [[alt.karp.orca]] [[alt.karp.orange]]",
   "Magikarp Orange Dapples": "[[Magikarp]] [[alt.karp.dapples]] [[alt.karp.orange]]",
   "Magikarp Pink Two Tone": "[[Magikarp]] [[alt.karp.twoTone]] [[alt.karp.pink]]",
```

I messed up resolving a conflict and forgot this one. Is a suggestion comment really the only way I can commit a change without pulling it down locally?
```suggestion
  "Magikarp Pink Two-Tone": "[[Magikarp]] [[alt.karp.twoTone]] [[alt.karp.pink]]",
```